src/hotspot/share/runtime/os.cpp

```diff
@@ -1176,12 +1176,12 @@ void os::print_location(outputStream* st, intptr_t x, bool verbose) {
 
 // Looks like all platforms can use the same function to check if C
 // stack is walkable beyond current frame.
+// Returns false if this is the cas
 bool os::is_first_C_frame(frame* fr) {
 
 #ifdef _WINDOWS
   return true; // native stack isn't walkable on windows this way.
-#endif
-
+#else
```

Comment:
This change has nothing to do with the bug.

I would leave this as it is and let the code below at least compile on windows. Then we know it does not bitrot there. I am also not clear why this would not work on windows, since we could optionally build with framepointers enabled, right? And don't we have frame pointers on 32-bit windows always? I may remember this wrong.

Reply:
There is a special function on windows to obtain native stack traces, it uses an OS function